tests/async.test.tsx

```diff
@@ -767,9 +767,7 @@ it('non suspense async write self atom with setTimeout (#389)', async () => {
 })
 
 it('should override promise as atom value (#430)', async () => {
-  const countAtom = atom(
-    new Promise<number>((r) => setTimeout(() => r(-1), 3600 * 1000))
```

Comment:
this means the promise should never resolve. sorry for the confusion.
let's make it `new Promise<number>)(() => {}) // infinite promise`.

Reply:
Fixed :)